### DISCUSSION TITLE
apple/swift-syntax conflicts with swiftlang/swift-syntax

### DISCUSSION THREAD

**Discussion by atacan** [pointfreeco/swift-composable-architecture#3262] [General]
I get this yellow warning in Xcode:

`
x-xcode-log://3AF43ED7-D11A-4F8B-8A37-EF3F764F706B 'swift-composable-architecture' dependency on 'https://github.com/apple/swift-syntax' conflicts with dependency on 'https://github.com/swiftlang/swift-syntax' which has the same identity 'swift-syntax'. this will be escalated to an error in future versions of SwiftPM.
`

My Package.swift looks like

```swift
// swift-tools-version: 5.7
// The swift-tools-version declares the minimum version of Swift required to build this package.

// ...

        .package(url: "https://github.com/pointfreeco/swift-composable-architecture", exact: "1.11.0"),
        .package(url: "https://github.com/pointfreeco/swift-dependencies", from: "1.1.0"),
```

**Comment by mbrandonw** [comment]
Hi @atacan, this is due to Apple moving swift-syntax to a different organization. You can either ignore the warning, or update your packages to the newest versions.
